README.md

```diff
@@ -57,7 +56,6 @@ failedTestToggle();
 module.exports = defineConfig({
```

Comment:
Just a nit not associated to these changes but can you include where `collectFailingTests` comes from? 

Do we need to `const { collectFailingTests } = require(cypress-plugin-last-failed` at the top of our config file to get that method?

Reply:
Thanks for pointing that out @davidmatium 🙏 Added your suggestion to the code example here in the README.

We need to import the method into the config here, which is stored in the main `index` of the package.